Makefile

```diff
@@ -190,6 +190,10 @@ generate: controller-gen
 	hack/update-openapigen.sh
 	hack/python-sdk/client-gen.sh
 
+bump-version:
+	# TBA
+	echo "bumping version numbers for this release"
```

Comment:
Only a placeholder for now. We are bumping the versions manually this time. https://github.com/kserve/kserve/pull/3352